examples/simple-scenario/.gitignore

```diff
@@ -0,0 +1,84 @@
+# Created by .ignore support plugin (hsz.mobi)
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion
+
+../../.idea/modules/akkadebugging-core-build.iml
```

Comment:
I think that this is connected with the fact with moving files to different folders.